actors/miner/src/lib.rs

```diff
@@ -1950,6 +1950,70 @@ impl Actor {
         Ok(ProveCommitSectors3Return { activation_results: result })
     }
 
+    fn confirm_sector_proofs_valid(
+        rt: &impl Runtime,
+        params: ConfirmSectorProofsParams,
+    ) -> Result<(), ActorError> {
+        rt.validate_immediate_caller_is(std::iter::once(&STORAGE_POWER_ACTOR_ADDR))?;
```

Comment:
Change validation to check that the caller is the SYSTEM_ACTOR.
This signals that this can only be called externally.